allauth/socialaccount/providers/dingtalk/client.py

```diff
@@ -0,0 +1,32 @@
+import requests
+
+from allauth.socialaccount.providers.oauth2.client import (
+    OAuth2Client,
+    OAuth2Error,
+)
+
+
+class DingTalkOAuth2Client(OAuth2Client):
+    def get_access_token(self, code):
+        data = {
```

Comment:
Can you have a look at the build -- it is failingg here.

Reply:
> Can you have a look at the build -- it is failingg here.

ok, `PKCE` support was added

Reply:
ok, PKCE support was added